src/python/motuclient.py

```diff
@@ -231,6 +227,23 @@ def load_options():
                        action='store_true',
                        dest='console_mode')
 
+    parser.add_option( '--config-file',
+                       help = f"Path of the optional configuration file [default: {CFG_FILE}]",
```

Comment:
We are merging your code and we have an issue while executing the code.
Please confirm that the 'f' in 
help = f"Path
is a mistake and that we have to remove it?

Reply:
Actually, this is a valid format called f-strings allowed since python 3.6 ([PEP 498](https://www.python.org/dev/peps/pep-0498/)). I use mostly python 3.7 and so I forgot that motuclient has to be compatible even with pyhon 2.7. You can just replace that line with:
```
 help = "Path of the optional configuration file [default: %s]"%(CFG_FILE),
```
Sorry for the trouble. I should have tested on python 2.7 also.

Reply:
Corrected the problematic line with a new commit.

Reply:
Thanks to have taken into account this compatibility with Python Python 2.7, we will integrate it and release a news version in the coming days.